### PR TITLE
benchmark fees

### LINKED ISSUE #356: Benchmarking Fees Pallet

### PR DESCRIPTION
**Pull request by vedhavyas**
This PR adds the benchmarking base for our pallets and Benchmarking fees pallet 

Closes #356

### CODE REVIEW

File: scripts/frame-weight-template.hbs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+			{{~/each}}
+			{{~#if (ne benchmark.base_writes "0")}}
+			.saturating_add(RocksDbWeight::get().writes({{benchmark.base_writes}} as Weight))
+			{{~/if}}
+			{{~#each benchmark.component_writes as |cw|}}
+			.saturating_add(RocksDbWeight::get().writes(({{cw.slope}} as Weight).saturating_mul({{cw.name}} as Weight)))
+			{{~/each}}
+	}
+	{{~/each}}
+}
```
Comment (mustermeiszer) [inline scripts/frame-weight-template.hbs:86]: Will this be our general template then?

**Review by mustermeiszer — APPROVED** [review]
Thanks Ved!

File: scripts/frame-weight-template.hbs (hunk 1 above)
Comment (vedhavyas) [reply scripts/frame-weight-template.hbs:86]: I like this since it Includes the Trait definition and Impls. Any concerns @mustermeiszer ?

Comment (mustermeiszer) [reply scripts/frame-weight-template.hbs:86]: No, I also like this! Was just curious.

**Review by mikiquantum — APPROVED** [review]
Awesome! Thanks @vedhavyas , indeed our weights are way too much.

**Pull request merged**